hydroflow_lang/src/graph/ops/mod.rs

```diff
@@ -9,9 +9,18 @@ use syn::{Expr, GenericArgument, Token};
 
```

Comment:
not reviewing this carefully on the assumption that changes from the previous PR review are being postponed